ProdDesProject/Group8
# Health bar

### Added imba Death Animation for Dudeman

**Testing**

- Check out the new death animation for dudeman :D

- When Dudeman reaches 0 hp, a death animation should be played. It should work no matter in which state the player dies (on the ground, in the air, etc.)

- Check also that when the death animation activates, you cannot do anything as the player (move, jump, etc.)
